client/elements/suttaplex/sc-suttaplex-list.html

```diff
@@ -155,6 +166,12 @@
                   return levels[difficulty];
               }
           }
+
+          _expansionLoaded() {
+              let result = {};
+              this.expansionReturns[0].forEach((item) => result[item.uid] = item.acro);
```

Comment:
Transforming the data here seems unnecessary - the backend should already return the correct form. We can use MERGE in arangoDB to return one object instead of an array of objects.